.github/workflows/_docs.yml

```diff
@@ -166,6 +166,7 @@ jobs:
             -e DOCS_TYPE \
             -e RUN_DOXYGEN \
             -e WITH_PUSH \
+            -e PYTORCH_USE_CXX_PYTREE=1 \
```

Comment:
The docs are built using C++ pytree? Since the default for pytree is Python pytree, I'd prefer the docs to be built with Python pytree

Reply:
Reverted.